downstream/modules/vault-aap/proc-vault-creating-custom-credential.adoc

```diff
@@ -0,0 +1,24 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="vault-creating-custom-credential"]
+
+= Creating a custom credential
+
+[role="_abstract"]
+
+{Vault} users must create a custom credential to use with job templates in {PlatformNameShort}.
+
+.Prerequisite
+* Your administrator has created a {Vault} credential type.
+
+.Procedure
+
+. Log in to {PlatformNameShort}.
+. From the navigation panel, select **Automation Execution > Infrastructure > Credentials** page, and select **Create credential**.
```

Comment:
Again, sorry for not pointing out the menu paths in our attributes doc. Similar to the above suggestion: 

```suggestion
. From the navigation panel, select {MenuAECredentials}, and select **Create credential**.
```